notebooks/book2/15/logreg_iris_bayes_2d.ipynb

```diff
@@ -1,49 +1,229 @@
 {
```

Comment:
<p>what is <code>x1_all</code>?Is it the same as <code>bd</code>,the decision boundary?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/probml/pyprobml/pull/994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
I've renamed it to `bd` and added a comment to avoid confusion.